ui/lib/core/addon/components/linked-block.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="linked-block" role="link" {{on "click" this.onClick}} ...attributes>
```

Comment:
Should this be `role="button"` or `role="link"` since we're doing a function action on click?

Reply:
:thinking:. I suppose button would be more appropriate. Although the click action results in a transition so in that way it behaves as a link. I really don't know where to land :smile:. What do you think?

Reply:
hmm.... I'm going to ask our resident a11y experts if they have an opinion on this but I think, to your point, since the behavior is to navigate it probably makes sense to stay a link